packages/package_info_plus/package_info_plus/example/android/build.gradle

```diff
@@ -1,29 +1,16 @@
-buildscript {
-    ext.kotlin_version = '1.7.22'
-    repositories {
-        google()
-        mavenCentral()
-    }
-
-    dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath 'com.android.tools.build:gradle:8.3.0'
-    }
-}
-
 allprojects {
     repositories {
         google()
         mavenCentral()
     }
 }
 
-rootProject.buildDir = '../build'
+rootProject.layout.buildDirectory = '../build'
```

Comment:
This is not related to the guidance that I shared, but something that was deprecated in Gradle 8.2.
https://docs.gradle.org/current/userguide/upgrading_version_8.html#project_builddir

Considering that we require 8.4 now due to Android Gradle Plugin 8.3.0, I thought that it won't hurt to address this deprecation as well.

Reply:
Reverted this change to stick with only changes from the guidance I shared. Hope it works now.

Reply:
Looks like after reverting this change all works as expected.